packages/perseus/src/analytics/types.ts

```diff
@@ -0,0 +1,6 @@
+import type {SendEvent} from "@khanacademy/perseus-core";
+
+/** An event sink for analytics events. */
+export type AnalyticsSink = {
```

Comment:
I debated just attaching the `SendEvent` directly to `PerseusDependencies`.  It feels kinda nice for it to be nested in an `analytics` key as it reads as `deps.analytics.sendEvent(...)` instead of just `deps.sendEvent(...)` (what kind of event??)

Reply:
I like the idea of having more structure in `PerseusDeps`, but I'd probably call it `Analytics` instead of `AnalyticsSink`.  The latter seems overly specific and likely to require renaming in future.